### PR TITLE
fix!: Use JSON tag for column names

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Related to https://github.com/cloudquery/cloudquery/issues/2189, extracted from https://github.com/cloudquery/cloudquery/pull/2623. Tests for code gen drift for the Cloudflare plugin.

This is a breaking change as it renames a few columns due to https://github.com/cloudquery/plugin-sdk/pull/112

### CODE REVIEW

File: plugins/source/cloudflare/codegen/main.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 	"strings"
 	"text/template"
 
-	"github.com/cloudquery/cloudquery/plugins/source/cloudflare/codegenmain/recipes"
+	"github.com/cloudquery/cloudquery/plugins/source/cloudflare/codegen/recipes"
```
Comment (erezrokah) [inline plugins/source/cloudflare/codegen/main.go:15]: Renamed this package and moved the files to be consistent with other plugins

File: plugins/source/cloudflare/codegen/recipes/account.go [hunk 2]
```diff
@@ -23,9 +23,9 @@ func AccountResources() []Resource {
 			CFStruct: &cloudflare.AccountMember{},
 			DefaultColumns: []codegen.ColumnDefinition{
 				{
-					Name:     "account_cq_id",
-					Type:     schema.TypeUUID,
-					Resolver: "schema.ParentIDResolver",
+					Name:     "account_id",
```
Comment (erezrokah) [inline plugins/source/cloudflare/codegen/recipes/account.go:26]: This was updated in the resources, but not in the code gen (I was the one who did this update and missed the code gen changes)

File: plugins/source/cloudflare/resources/services/images/images.go [hunk 3]
```diff
@@ -38,7 +38,7 @@ func Images() *schema.Table {
 				Resolver: schema.PathResolver("Metadata"),
 			},
 			{
-				Name:     "require_signed_ur_ls",
+				Name:     "require_signed_urls",
```
Comment (erezrokah) [inline plugins/source/cloudflare/resources/services/images/images.go:41]: Breaking change

File: plugins/source/cloudflare/resources/services/worker_meta_data/workers_secrets.go [hunk 4]
```diff
@@ -22,7 +22,7 @@ func workersSecrets() *schema.Table {
 				Resolver: schema.PathResolver("Name"),
 			},
 			{
-				Name:     "type",
+				Name:     "secret_text",
```
Comment (erezrokah) [inline plugins/source/cloudflare/resources/services/worker_meta_data/workers_secrets.go:25]: Breaking change

File: plugins/source/cloudflare/resources/services/zones/zones.go [hunk 5]
```diff
@@ -33,12 +33,12 @@ func Zones() *schema.Table {
 				Resolver: schema.PathResolver("Name"),
 			},
 			{
-				Name:     "dev_mode",
+				Name:     "development_mode",
```
Comment (erezrokah) [inline plugins/source/cloudflare/resources/services/zones/zones.go:36]: All breaking changes due to taking the column names from JSON tags

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**